docs/reference/cheatsheet.md

```diff
@@ -0,0 +1,241 @@
+# ehrQL cheatsheet
+
+## Frames
+
+Patient frames contain one row per patient
+
+```
+patient id date_of_birth sex
+123        1980-01-01    m
+456        1990-06-06    f
+789        2020-01-01    i
+```
+
+Event frames contain many rows per patient
+
+```
+patient_id event_date event_code
+123        2020-04-01 abc
+123        2020-04-02 def
+123        2021-01-01 ghi
+```
+
+## Simple dataset
+
+```python
+from ehrql import create_dataset
+from ehrql.tables.core import patients
+from ehrql.tables.tpp import addresses
+
+dataset = create_dataset()
+...
+dataset.define_population(
+    patients.exists_for_patient()
+)
+```
+
+## Codelists
+
+```python
+statin_medications = codelist_from_csv("XXX", column="YYY")
+```
+
+## Debug
+
+```python
+from ehrql import debug
+debug(patients)
+```
+
+## Tables
+
+### core
+
+```python
+clinical_events
+medications
+ons_deaths
+patients
+practice_registrations
+```
+
+#### selected values
+
+```python
+clinical_events.date
+clinical_events.snomedct_code
+clinical_events.numeric_value
+patients.date_of_birth
+patients.sex
+patients.date_of_death
+```
+
+### tpp
```

Comment:
yeah, it depends on format - I started with a markdown file like this & had the same thought as you, then I switched to a printed page layout & thought it would be useful, now it's back as a long markdown page it seems less valuable, I'd like to leave it for now & maybe reformat the page to more of the sort of shape that I'd expect from a cheatsheet (the main criteria being that a large proportion of the content is visible simultaneously)